README.md

```diff
@@ -0,0 +1,23 @@
+# Github Views Logger  
+This is a mix of java and batch scripting that gets pertinent data from a user's repos and stores them.   
+It is set up to get and store a day-by-day count of views and clones that does not simply go away after 24 hours.  
+I'm not a superhero that can retrieve data that is more than 2 weeks old unless it has already been stored by the program, then it merely keeps going from the first time logging started.  
+
+## REQUIRED STEPS:  
+go into ViewsGetter.bat and in the quotes, add a username, and the password (see note at end)  
```

Comment:
change this when we change the windows file

Reply:
should i make 2 sections for linux and windows?

Reply:
yeah but we can do that once we finish this merge thing